src/components/LocationSelector/LocationSelectorDialog/LocationSelectorDialog.js

```diff
@@ -217,6 +222,7 @@ LocationSelectorDialog.propTypes = {
     getMarkers: PropTypes.func.isRequired,
     /** function to be called when teh Done button is clicked */
     onCloseModal: PropTypes.func,
+    defaultHighlight: PropTypes.object,
```

Comment:
What's the difference with `defaultArea`?

Reply:
I added the description in Map (will add it everywhere if the direction is approved)
```
    /** The default parameters to determine the viewport when no markers are present. */
    defaultArea: PropTypes.oneOfType([
        PropTypes.shape({
            address: PropTypes.string,
        }),
        PropTypes.shape({
            center: PropTypes.oneOfType([
                PropTypes.arrayOf(PropTypes.number),
                PropTypes.shape({
                    lng: PropTypes.number.isRequired,
                    lat: PropTypes.number.isRequired,
                }),
            ]),
            zoom: PropTypes.number,
        }),
    ]),
    /** A geoJson description of the area that should be highlighted when there are no other markers present */
    defaultHighlight: PropTypes.array,
```